ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,175 @@
+    color: f9f2f2
+  - name: B5-unused_host_function_breaking_change
+    description: PR introduces a breaking change to the behavior of a host function which is not used by any runtime.
+    color: f9f2f2
+  - name: B6-remove_unused_host_function
+    description: PR removes a host function not used by any runtime.
+    color: f9f2f2
+  - name: B7-breaking_change_RPC_API
+    description: This PR this changes the behavior of the external RPC API in a backwards incompatible way.
+    color: cdadad
+  - name: C0-easy
+    description: Can be fixed primarily by duplicating and adapting code by an intermediate coder.
+    color: eeeeee
+  - name: C1-medium
+    description: Can be fixed by a coder with good Rust knowledge but little knowledge of the codebase.
+    color: eebb00
+  - name: C2-substantial
+    description: Can be fixed by an experienced coder with a working knowledge of the codebase.
+    color: D4C5F9
+  - name: C3-involved
+    description: Can be fixed by an expert coder with good knowledge of the codebase.
+    color: E99695
+  - name: C4-mentor
+    description: An easy task where a mentor is available. Please indicate in the issue who the mentor could be.
+    color: "5319e7"
+  - name: C5-reserved
+    description: This issue should not be picked up by anyone unless approved.
+    color: cdadad
+  - name: D0-nice_to_have_audit
+    description: PR contains trivial changes to logic that should be properly reviewed.
+    color: c5def5
+  - name: D1-needs_audit
+    description: PR contains changes to fund-managing logic that should be properly reviewed and externally audited.
+    color: C3EDC0
+  - name: D2-needs_DevRel_audit
+    description: PR contains changes that are relevant for documentation, should be audited by DevRel before merge.
+    color: C0E7CC
+  - name: D3-audited
+    description: PR contains changes to fund-managing logic that has been properly reviewed and externally audited.
+    color: C0E7CC
+  - name: I0-panic
+    description: The node panics and exits without proper error handling.
+    color: 17b25d
+  - name: I1-security
+    description: The node fails to follow expected, security-sensitive, behaviour.
+    color: d93f0b
+  - name: I2-bug
+    description: The node fails to follow expected behavior.
+    color: "249899"
+  - name: I3-annoyance
+    description: The node behaves within expectations, however this “expected behaviour” itself is at issue.
+    color: D6075D
+  - name: I4-refactor
+    description: Code needs refactoring.
+    color: 0075ca
+  - name: I5-enhancement
+    description: An additional feature request.
+    color: c2e0c6
+  - name: I6-meta
+    description: A specific issue for grouping tasks or bugs of a specific category.
+    color: a2eeef
+  - name: I7-duplicate
+    description: Issue is a duplicate. Closer should comment with a link to the duplicate.
+    color: D62209
+  - name: I8-wont_fix
```

Comment:
This one also look like Action label.

Reply:
I initially had the same thought, but I see this as something similar to invalidating (`I9-invalid`) an issue. Obviously nobody would create an issue and label it as 'wont-fix/invalid' but I see it as something intrinsic to to the issue rather than to it's state.
Though, we could probably merge `wont-fix` and `invalid`, and then the reason in the comment would say more about it

Reply:
I will change the description of `won't fix` and keep only this one. The closer should then give further explanation in the comment